### ISSUE TITLE
Staging Review finding: Accessible name doesn't match visible label

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98445]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Identity
**Product:** sign-in transition
**Feature:** interstitial

## Findings details
**VA.gov Experience Standard - issue:** [User can't complete a necessary flow or hits a dead end.](https://design.va.gov/about/experience-standards/#user-can-complete-necessary-flows)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

Both the Login.gov and ID.me buttons are visually rendered with SVGs of the authentication service logos for their button text, and with an `aria-labelledby` pointing to the  `title` of the SVG to provide an accessible name. But both also contain `sr-only` text that reads "Verify with [auth service]."

In the accessibility tree, that makes the name of the button "Verify with Login.gov Login.gov" (and similar for ID.me).

For screen reader users, this is repetitive and possibly confusing ("verify" doesn't have a lot of context for what you're doing here). For voice command/speech recognition users, this is potentially a barrier to using the button. If a sighted voice user speaks the command "Click Login.gov," some software will be smart enough to find the match with "Verify with Login.gov Login.gov" (like Dragon on Windows), and some... won't (like Apple Voice Control).

<details><summary>Link, screenshot or steps to recreate</summary>

Example of a button as currently coded:
```
<button type="button" class="usa-button logingov-verify-buttons">
   <span class="sr-only">Verify with Login.gov</span>
   <svg role="img" ... aria-labelledby="logingov-title">   
       <title id="logingov-title">Login.gov</title>
       ...
   </svg>
</button>
```

</details>

## Recommended action

I don't think the `span` with the `sr-only` text is necessary! I think you can safely remove it.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Voice
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by claytonzook** [comment]
This was resolved prior to going live in production
